security/advancedtls/examples/go.mod

```diff
@@ -5,5 +5,10 @@ go 1.15
 require (
 	google.golang.org/grpc v1.33.1
 	google.golang.org/grpc/examples v0.0.0-20201020200225-9519efffeb5d
-	google.golang.org/grpc/security/advancedtls v0.0.0-20201020200225-9519efffeb5d
+	google.golang.org/grpc/security/advancedtls v0.0.0-20201102215344-15ae9fc2b247
```

Comment:
This should also get a `replace` directive, so we know the examples are in sync with HEAD.

Reply:
Done. Could you please take a look again? Thank you so much!